easybuild/easyconfigs/l/lagrangian-filtering/lagrangian-filtering-0.8.3-foss-2022a.eb

```diff
@@ -0,0 +1,83 @@
+# This installation unfortunately requires the GitHub dance, as the tarball
+# is simply not installing. For that reason there is no checksum! 
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'lagrangian-filtering'
+version = '0.8.3'
+
+homepage = 'https://github.com/angus-g/lagrangian-filtering'
+description = """Temporal filtering of data in a Lagrangian frame of reference.""" 
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('dask', '2022.10.2'),
+    ('h5py', '3.7.0'),
+    ('SciPy-bundle', '2022.05'),
+    ('netcdf4-python', '1.6.1'),
+    ('scikit-learn', '1.1.2'),
+    ('tqdm', '4.64.0'),
+    ('zarr', '2.13.3'),
+]
+
+use_pip = True
+sanity_pip_check = True
+
+exts_list = [
+    ('parcels', '2.4.0', {
+        'checksums': ['221ce49b6cb5bb4996aa3c817c0de7f022c952a4d8cb4160ca7df7dd940714b5'],
+    }),
+    ('xarray', '2022.9.0', {
```

Comment:
we have easyconfigs for [xarray](https://github.com/easybuilders/easybuild-easyconfigs/tree/develop/easybuild/easyconfigs/x/xarray), please change this extension into a regular dependency